eng/pipelines/pipeline-generation.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     env:
       PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
   - script: |
-      $(Pipeline.Workspace)/pipeline-generator/pipeline-generator --organization https://dev.azure.com/azure-sdk --project internal --prefix $(Prefix) --path $(Pipeline.Workspace)/$(RepositoryName)/sdk --endpoint Azure --repository Azure/$(RepositoryName) --convention up --agentpool Hosted --branch refs/heads/$(Branch) --patvar PATVAR --debug $(InternalOptions) $(TestOptions)
+      $(Pipeline.Workspace)/pipeline-generator/pipeline-generator --organization https://dev.azure.com/azure-sdk --project internal --prefix $(Prefix) --path $(Pipeline.Workspace)/$(RepositoryName)/sdk --endpoint Azure --repository Azure/$(RepositoryName) --convention up --agentpool Hosted --branch refs/heads/$(Branch) --patvar PATVAR --debug $(InternalOptions)
```

Comment:
@weshaggard I added `TestOptions` here because in order to enable smoke tests on release pipelines, the ci pipeline needs to be given access to the subscription configuration secrets, among others.

Reply:
I didn't realize it would break other pipelines, it's hard to test this exhaustively without just going in and making a bunch of pipeline changes before review :)

Reply:
@weshaggard fix: https://github.com/Azure/azure-sdk-tools/pull/1312

Reply:
Removed this change.